lmms_eval/tasks/voicebench/utils.py

```
@@ -0,0 +1,1547 @@
+                        audio_array = decoded_audio.data
+                    else:
+                        audio_array = decoded_audio
+
+                    if hasattr(audio_array, "cpu") and hasattr(audio_array, "numpy"):
+                        audio_array = audio_array.cpu().numpy()
+                    elif hasattr(audio_array, "detach"):
+                        audio_array = audio_array.detach().cpu().numpy()
+                    elif str(type(audio_array).__name__) == "Tensor":
+                        try:
+                            audio_array = audio_array.cpu().numpy()
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)